views/listing.php

```diff
@@ -185,101 +199,109 @@
-          if ($total_ratings > 0) {
-            while ($rating = $result_ratings->fetch_assoc()) {
-              echo(
-                '<p>Rating: ' 
-                . number_format($rating['rating'], 1) 
-                . '/5.0 - Submitted: ' 
-                . date('j M Y, g:ia', strtotime($rating['submittedTime'])) . '</p>');
-              echo('<p>Comments: ' . htmlspecialchars($rating['comment']) . '</p>');
-              echo('<hr>');
-            }
-          } else {
-            echo('<p>No ratings yet.</p>');
+        $stmt_ratings = $db->prepare($query_ratings);
+        $stmt_ratings->bind_param("i", $auction['sellerId']);
+        $stmt_ratings->execute();
+        $result_ratings = $stmt_ratings->get_result();
+        $total_ratings = $result_ratings->num_rows;
+        if ($total_ratings > 0) {
+          while ($rating = $result_ratings->fetch_assoc()) {
+            echo (
+              '<p>Rating: '
+              . number_format($rating['rating'], 1)
+              . '/5.0 - Submitted: '
+              . date('j M Y, g:ia', strtotime($rating['submittedTime'])) . '</p>');
+            echo ('<p>Comments: ' . htmlspecialchars($rating['comment']) . '</p>');
+            echo ('<hr>');
           }
+        } else {
+          echo ('<p>No ratings yet.</p>');
+        }
         ?>
+      </div>
     </div>
   </div>
-</div>
-<!-- End modal -->
+  <!-- End modal -->
 
-<?php include_once("footer.php") ?>
+  <?php include_once("footer.php");
+  ?>
 
-<script>
-  // JavaScript functions: addToWatchlist and removeFromWatchlist.
+  <script>
+    // JavaScript functions: addToWatchlist and removeFromWatchlist.
 
-  function addToWatchlist(button) {
-    console.log("These print statements are helpful for debugging btw");
+    function addToWatchlist(button) {
+      console.log("These print statements are helpful for debugging btw");
 
-    // This performs an asynchronous call to a PHP function using POST method.
-    // Sends item ID as an argument to that function.
-    $.ajax('watchlist_funcs.php', {
-      type: "POST",
-      data: {
-        functionname: 'add_to_watchlist',
-        arguments: [<?php echo ($item_id); ?>]
-      },
+      // This performs an asynchronous call to a PHP function using POST method.
+      // Sends item ID, user ID, and followerRows as arguments to that function.
+      $.ajax('watchlist_funcs.php', {
+        type: "POST",
+        data: {
+          functionname: 'add_to_watchlist',
+          argument_1: <?php echo $item_id; ?>,
+          argument_2: <?php echo $user_id; ?>,
+          argument_3: <?php echo $followerRows; ?>
+        },
 
-      success: function(obj, textstatus) {
-        // Callback function for when call is successful and returns obj
-        console.log("Success");
-        var objT = obj.trim();
+        success: function(obj, textstatus) {
+          // Callback function for when call is successful and returns obj
+          console.log("Success");

```

Comment:
I would usually try to remove these console logs after finishing development

Reply:
Ok, they're gone